Code/GraphMol/Depictor/RDDepictor.h

```diff
@@ -149,17 +156,56 @@ RDKIT_DEPICTOR_EXPORT unsigned int compute2DCoordsMimicDistMat(
   \param referencePattern -  (optional) a query molecule to be used to
                              generate the atom mapping between the molecule
                              and the reference.
-  \param acceptFailure - (optional) if True, standard depictions will be
-  generated
-                         for molecules that don't have a substructure match to
-  the
-                         reference; if false, throws a DepictException.
+  \param acceptFailure - (optional) if true, standard depictions will be
+                         generated for molecules that don't have a substructure
+                         match to the reference; if false, throws a
+                         DepictException.
+  \param forceRDKit - (optional) use RDKit to generate coordinates even if
+                      preferCoordGen is set to true
+  \param allowRGroups -  (optional) if true, terminal dummy atoms in the
+                         reference are ignored if they match an implicit
+                         hydrogen in the molecule, and a constrained
+                         depiction is still attempted
 
 */
-RDKIT_DEPICTOR_EXPORT void generateDepictionMatching2DStructure(
+RDKIT_DEPICTOR_EXPORT RDKit::MatchVectType generateDepictionMatching2DStructure(
```

Comment:
Please document the return type of this function.

Reply:
It seems documented to me; what am I missing?

Reply:
where is it documented? I don't see it in this comment:
https://github.com/ptosco/rdkit/blob/allowRGroups/Code/GraphMol/Depictor/RDDepictor.h#L140

Reply:
Oh, crap, I put in the wrong comment :) I'll fix that now.
https://github.com/ptosco/rdkit/blob/6f09b2a808f41ee9d82394110b519a1f6a28d708/Code/GraphMol/Depictor/RDDepictor.h#L202